Unit test fixes

dilinade opened the pull request:
Unit tests have been fixed after adding new parameters. (email,drone description etc)

![image](https://user-images.githubusercontent.com/14290956/53778849-36aa1700-3f24-11e9-9c68-2c79fdd8a819.png)

**Closing issues**

Closes #179  #114

meetcric commented:
Hello @dilinade ,It will be better for review purpose if you Squash multiple commits to a single commit.

dilinade committed:
Modified tests

Added travis.yml

Modified db file with travis

Fixes codacy checks

Update example.db.js
Update example.db.js
Update .travis.yml
Update .travis.yml
Update .travis.yml
Update .travis.yml
Update .travis.yml
Update .travis.yml
Update .travis.yml

dilinade committed:
Update .travis.yml

dilinade commented:
@meetcric Thank you. Didn't notice. I'm doing it.

ivantha commented:
@dilinade Thank for the PR 🙂
